components/Graph/NodeModal/NodeModal.tsx

```diff
@@ -1,78 +1,67 @@
 import React from "react"
-import { Modal, TouchableWithoutFeedback, View, Text } from "react-native"
-import Svg, { Image } from "react-native-svg"
+import {
+  Modal,
+  TouchableWithoutFeedback,
+  View,
+  Text,
+  Image,
+} from "react-native"
 import { Node } from "../Graph"
 
 import styles from "./styles"
 
-import profile_picture_0 from "../../../assets/graph-template-profile-pictures/graph-template-profile-picture-0.png"
-import profile_picture_1 from "../../../assets/graph-template-profile-pictures/graph-template-profile-picture-1.png"
-import profile_picture_2 from "../../../assets/graph-template-profile-pictures/graph-template-profile-picture-2.png"
-import profile_picture_3 from "../../../assets/graph-template-profile-pictures/graph-template-profile-picture-3.png"
-import profile_picture_4 from "../../../assets/graph-template-profile-pictures/graph-template-profile-picture-4.png"
-import profile_picture_5 from "../../../assets/graph-template-profile-pictures/graph-template-profile-picture-5.png"
-import profile_picture_6 from "../../../assets/graph-template-profile-pictures/graph-template-profile-picture-6.png"
-
-const PROFILE_PICTURES = [
-  profile_picture_0,
-  profile_picture_1,
-  profile_picture_2,
-  profile_picture_3,
-  profile_picture_4,
-  profile_picture_5,
-  profile_picture_6,
-]
-
+/**
+ * NodeModal component
+ * @param node - The node to display
+ * @param visible - Whether the modal is visible
+ * @param onPressOut - Function to call when the modal is pressed out
+ * @param onContactPress - Function to call when the contact is pressed
+ * @returns The NodeModal component
+ */
```

Comment:
I don't think it is really necessary to add documentation for components - especially when the prop variable names are already intuitive enough

Reply:
Okay good points!